implementations/rust/ockam/ockam_app/src/invitations/plugin.rs

```diff
@@ -0,0 +1,39 @@
+use tauri::{
+    plugin::{Builder, TauriPlugin},
+    Manager, Runtime,
+};
+
+use super::{
+    commands::*,
+    events::{REFRESHED_INVITATIONS, REFRESH_INVITATIONS},
+};
+
+pub(crate) fn init<R: Runtime>() -> TauriPlugin<R> {
```

Comment:
I like this approach! Didn't see plugins in action yet

Reply:
It's been working relatively well so far, and I think offered slightly more utility during my initial design where the State was totally separated. You can kind of see that in action in this commit range, before I collapsed it into AppState:

c1326faa8525e836cb756ce429bb5607b9bacd68...72a5b9b424df6e63c2269b50c3a3bbf2b4f91384